DEVELOPMENT.md

```diff
@@ -10,6 +10,12 @@ When working on compliantkubernetes-apps it is possible to work with local clust
 
 This requires that `kind` is installed and that either `podman` or `docker` is available and setup to work with `kind`.
 
+### Terminology
+
+- `apps-flavor` - one of `prod`, `dev` or `baremetal` is a global switch for configuring the clusters and deployed applications in different ways. The recommended value when working with local clusters is `dev`.
+- `local-cluster-profile` - a `Cluster` configuration object passed to `kind` when creating or updating local clusters. We use profiles to differentiate between single-/multi-node clusters and/or to enable special features such as container image caching. Use `./bin/local-clusters.sh list profiles` to see a list of built-in profiles.
```

Comment:
To be pedantic (as it is written `<kind-config|local-cluster-profile>` in usage latter), the profiles as the preconfigured `Cluster` configs.